### ISSUE TITLE
Documentation?

### ISSUE THREAD

**Issue by wmayner** [biocore-ntnu/ncls#27]
I just discovered this library and it looks excellent. Because the functions aren't native Python, there aren't any docstrings to document them via normal Python introspection. It would be helpful if there were API documentation in addition to the usage examples in the README 😄

**Comment by endrebak** [comment]
Yes, I am open to PRs. Will write better docs for the pyrangesverse when I finish my PhD. Keeping this open as a reminder :)
